cachito/web/config.py

```diff
@@ -19,4 +19,10 @@ class DevelopmentConfig(Config):
 
 class TestingConfig(DevelopmentConfig):
     """The testing Cachito Flask configuration."""
-    pass
+    # IMPORTANT: don't use in-memory sqlite. Alembic migrations will create a new
+    # connection producing a new instance of the database which is deleted immediately
+    # after migration completes...
```

Comment:
`after migration` => `after the migration`

Reply:
Done!